bluebrain/repo-bluebrain/packages/py-visual-cortex-analysis/package.py

```diff
@@ -12,7 +12,7 @@ class PyVisualCortexAnalysis(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis"
     git = "ssh://git@bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis.git"
 
-    version("0.2.2", tag="visual-cortex-analysis-v0.2.2")
+    version("0.2.3", tag="visual-cortex-analysis-v0.2.3")
```

Comment:
Is that tag present in the repository?

Reply:
Hi Matthias, no, the job to release that version failed. It should be up in a few minutes.

Reply:
Hi @matz-e  , the new version is finally out, can you re-run the CI?